### PR TITLE
Fix erroneous license violation in LICENSE

### PR DESCRIPTION
**Pull request by mackmudge**
Removing the original "Copyright (c) 2023, Team Vendetta" is in violation of the license, the correct way is to add a second "Copyright (c) 2024, Team Sunset" line below the original. This fixes it.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,6 @@
 BSD 3-Clause License
 
+Copyright (c) 2024, Team Vendetta
```
Comment (mufaroxyz) [inline LICENSE:3]: Shouldn't it be 2023? In Vendetta's LICENSE file it's 2023 and they're not maintaining it anymore. Maybe I'm not understanding it correctly though

Comment (otomir23) [reply LICENSE:3]: Yes, you're correct, it's the year that copyright was applied, so it should be 2023

Comment (mackmudge) [reply LICENSE]: Yes my apologies I did this a little rushed, it should be. Have edited.

**Review by voofing — APPROVED** [review]
Seems good to me.

**Pull request merged**